### DISCUSSION TITLE
Install Playbook on IMPORT build of SO

### DISCUSSION THREAD

**Discussion by mmasino-matc** [Security-Onion-Solutions/securityonion#2711] [Unsupported Versions]
Is there a way to install Playbook on the IMPORT install of security onion? I want to use it to develop rules offline for imported logs. I could be missing to point of the IMPORT build also. Is doing a standard build and connecting the interface to a no traffic interface be a better way to do this?

**Comment by weslambert** [comment]
We don't currently provide this as an option for the import node.  You could install an eval/standalone build for the time being.
